src/components/TableModule/TableModule.tsx

```diff
@@ -249,7 +256,7 @@ export interface TableModuleProps<Item = any>
  * - [Story Source](https://github.com/lifeomic/chroma-react/blob/master/stories/components/TableModule/TableModule.stories.tsx)
  */
 export const TableModule = React.memo(
-  React.forwardRef<HTMLTableElement, TableModuleProps>(
+  React.forwardRef<HTMLTableElement, TableModuleProps<any>>(
```

Comment:
Just want to make sure this <any> is the best course of action here

Reply:
I remove this change but need such change in test code